drivers/bridge/setup_ipv6.go

```diff
@@ -133,17 +132,12 @@ func setupNDPProxying(config *networkConfiguration, i *bridgeInterface) error {
 			logrus.Warnf("Unable to enable NDP default proxying: %v", err)
 		}
 	}
-
-	// Get current NDP all proxying setup
-	ndpProxyDataAll, err := ioutil.ReadFile(ndpProxyConfAll)
```

Comment:
Yeah, to follow same logic we do for the default, for example

https://github.com/mostolog/libnetwork/blob/907282e5195e7bfaf507da0375bcaaf95a618603/drivers/bridge/setup_ipv6.go#L125

First read the value, then set it if not already set.